Bonsai.Configuration/ScriptExtensionsProjectMetadata.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using System.Xml.XPath;
+
+namespace Bonsai.Configuration
+{
+    public readonly struct ScriptExtensionsProjectMetadata
+    {
+        private readonly XDocument projectDocument;
+
+        public bool Exists => projectDocument is not null;
+
+        internal const string ItemGroupElement = "ItemGroup";
+        internal const string PackageReferenceElement = "PackageReference";
+        internal const string PackageIncludeAttribute = "Include";
+        internal const string PackageVersionAttribute = "Version";
+        internal const string UseWindowsFormsElement = "UseWindowsForms";
+        internal const string AllowUnsafeBlocksElement = "AllowUnsafeBlocks";
+
+        internal ScriptExtensionsProjectMetadata(XDocument projectDocument)
+            => this.projectDocument = projectDocument;
+
+        private XElement GetProperty(string key)
+            => projectDocument?.XPathSelectElement($"/Project/PropertyGroup/{key}");
+
+        private bool GetBoolProperty(string key)
+            => String.Equals(GetProperty(key)?.Value, "true", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
```suggestion
            => string.Equals(GetProperty(key)?.Value, "true", StringComparison.InvariantCultureIgnoreCase);
```

Nitpick, just for consistency with the rest of the codebase.

Reply:
Coulda sworn I fixed that, thanks! Added the rule to the editorconfig too.